scripts/deploy.js

```diff
@@ -1,7 +1,7 @@
 let shell = require(`shelljs`);
 let pathToRepo = require(`path`).resolve(`.`);
 
-if(process.env.TRAVIS_PULL_REQUEST) {
+if(process.env.TRAVIS_PULL_REQUEST !== "false") {
```

Comment:
unfortunately no, because it won't be `"true"` it tends to be a number I think which will be hard to test (again because I don't know what type it is)